### PR TITLE
Update README to include the deploying document

### PR DESCRIPTION
**Pull request by JavierJia**
This PR adds the documentation how to deploy the Cloudberry and TwitterMap server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,8 +61,21 @@ There should be one zip file called `neo-1.0-SNAPSHOT.zip` generated under `clou
...
 ```
 
+The `application.secret` value should be the same with the one in the application.conf during `sbt dist`. You can find more in Play! framework [documentation page](https://www.playframework.com/documentation/2.6.x/Deploying)
+
 When it runs, there will be one `RUNNING_PID` file generated that includes the PID of the instance. You can kill the corresponding process to stop the instance. 
 
+You can run the *TwitterMap* application in the same way by following command:
+```
+> cd examples/twittermap
+> sbt "project neo" "clean" "dist"
```
Comment (waans11) [inline README.md]: sbt "project web" "clean" "dist" ?

Comment (JavierJia) [reply README.md]: done.

File: README.md [hunk 2]
```diff
@@ -49,3 +49,33 @@ You can run the following command in a separate command line window.
...
+
+There should be one zip file called `neo-1.0-SNAPSHOT.zip` generated under `cloudberry/neo/target/universal/`. 
+
+You can copy the file to where you want to run the Cloudberry, unzip it, and run the instance in the background as follows:
+```
+> cd neo-1.0-SNAPSHOT
+> bin/neo -Dapplication.secret='Yf]0bsdO2ckhJd]^sQ^IPISElBrfy<XWdTWukRwJK8KKc3rFG>Cn;nnaX:N/=R1<' -Dconfig.file=/full/path/to/production.conf  &
+```
+
+The `application.secret` value should be the same with the one in the application.conf during `sbt dist`. You can find more in Play! framework [documentation page](https://www.playframework.com/documentation/2.6.x/Deploying)
```
Comment (chenlica) [inline README.md]: "same with" -> "same as"

Comment (chenlica) [inline README.md]: A different word for "during"?

Comment (chenlica) [reply README.md]: "in Play! " -> "in the Play! "

File: README.md [hunk 3]
```diff
@@ -49,3 +49,33 @@ You can run the following command in a separate command line window.
...
+
+When it runs, there will be one `RUNNING_PID` file generated that includes the PID of the instance. You can kill the corresponding process to stop the instance. 
+
+You can run the *TwitterMap* application in the same way by following command:
```
Comment (chenlica) [inline README.md]: "by following command" -> "by running the following command"

File: README.md (hunk 2 above)
Comment (JavierJia) [reply README.md]: done

Comment (JavierJia) [reply README.md]: done.

File: README.md (hunk 3 above)
Comment (JavierJia) [reply README.md]: done.

**Pull request merged**